### PR TITLE
[WIP] first try at fixing #7 - notnamespaced resources

### PR DESCRIPTION
**Pull request by eyenx**

### CODE REVIEW

File: src/back8sup.sh [hunk 1]
```diff
@@ -32,6 +32,7 @@ readonly DST_FOLDER=${DST_FOLDER:-/mnt/back8sup}
 readonly CONFIGMAP_PATH=${CONFIGMAP_PATH:-/etc/config.yaml}
 readonly EXPORT_FORMAT=${EXPORT_FORMAT:-yaml}
 readonly BINARIES="kubectl yq jq yamllint"
+readonly NOTNAMESPACEDDIR=${NOTNAMESPACEDDIR:-not-namespaced}
```
Comment (hairmare) [inline src/back8sup.sh:35]: Did you consider defaulting this to `kubectl api-resources --namespaced=false`?

Comment (eyenx) [reply src/back8sup.sh:35]: this is the destination dir for the notnamespaced resources, why should it default to `kubectl api-resources --namespaced=false` ?

File: src/back8sup.sh [hunk 2]
```diff
@@ -108,6 +120,7 @@ do
     done
   done
   log "INFO done exporting all $KIND"
+  fi
```
Comment (hairmare) [inline src/back8sup.sh:123]: dat fi tho

Comment (eyenx) [reply src/back8sup.sh:123]: is it wrong?

**Pull request merged**